defaults/test_default.yaml

```diff
@@ -76,5 +76,5 @@ region_aware_loader: false
 stop_test_on_stress_failure: true
 
 stress_cdc_log_reader_batching_enable: true
-use_legacy_cluster_init: true
```

Comment:
This is not what I meant.
If the default was already "false", I don't understand why you needed to set it to "false" in the 1tb test.
(your original PR).